index.html

```diff
@@ -2286,10 +2286,13 @@ <h3>Presentations</h3>
           <dt><var id="defn-verifiableCredential">verifiableCredential</var></dt>
           <dd>
 The <code>verifiableCredential</code> <a>property</a> MAY be present. The value
-MUST be an array of one or more <a href="#verifiable-credential-graphs">
-verifiable credential graphs</a> in a cryptographically <a>verifiable</a> format.
-See Section <a href="#verifiable-credential-graphs"></a> for further details on
-this topic.
+MUST be one or more <a>verifiable credential</a> and/or
+<a href="#enveloped-verifiable-credentials">enveloped verifiable credential</a>
+objects. For the avoidance of doubt, the values MUST NOT be non-object values such as
+numbers, strings, or URLs. These types of objects are called
+<a href="#verifiable-credential-graphs">verifiable credential graphs</a> and
+MUST be secured using a <a href="#securing-mechanisms">securing mechanism</a>.
```

Comment:
Hmm, an enveloped verifiable credential is not itself secured, rather its contents are. With the current language, it would need to be enveloped again (and then this problem infinitely recurses), whereas a VC with an embedded proof is secured. To try and cover both of these cases, I propose this slight tweak here:

```suggestion
<a href="#verifiable-credential-graphs">verifiable credential graphs</a> and
MUST express information that is secured using a
<a href="#securing-mechanisms">securing mechanism</a>.
```